tensorflow_addons/utils/test_utils.py

```diff
@@ -26,7 +26,9 @@
 from tensorflow_addons.utils import resource_loader
 
 # TODO: copy the layer_test implementation in Addons.
-if tf.__version__[:3] > "2.5":
+if tf.__version__[:3] > "2.8":
+    from keras.testing_infra.test_utils import layer_test  # noqa: F401
+elif tf.__version__[:3] > "2.5":
```

Comment:
We don't need this condition anymore.

Reply:
Yes probably we have code from different epochs

https://github.com/tensorflow/addons/search?q=packaging.Version&type=